.github/workflows/build-addon-on-push.yml

```diff
@@ -90,8 +90,14 @@ jobs:
       - name: Create Godot-CPP library
         run: |
           cd aar/thirdparty/godot-cpp
-          scons platform=${{ matrix.platform }} target=template_debug ${{ matrix.flags }}
-          scons platform=${{ matrix.platform }} target=template_release ${{ matrix.flags }}
+          scons platform=${{ matrix.platform }} target=template_debug arch=arm64 
```

Comment:
This should be handled through the matrix, adding the different variations. Not all platforms support these archs, so this will break on MacOS

Reply:
Ah missed that, sorry :) that makes perfect sense!